packages/text-bitmap/src/BitmapFont.ts

```diff
@@ -18,15 +21,18 @@ interface IBitmapFontCharacter {
 // Internal map of available fonts, by name
 const available: Dict<BitmapFont> = {};
 
+// Default dimensions of the bitmap font atlas texture
+const BMT_DIMEN = 1024;
```

Comment:
Is there a reason the default size is 1024 and note 2048, since all browsers support 2048x2048 textures? Or is it because it's likely for general usage 1024x1024 will cover the glyphs required for English?

Reply:
Seems like an arbitrary-ish magic number, right @SukantPal?

Reply:
@bigtimebuddy Correct. Should I add an option for this in `IBitmapFontOptions`.

Reply:
I added `textureWidth`, `textureHeight` to `IBitmapFontOptions`. Also, `BMT_DIMEN` is 2048 as suggested by Dave.